backend/manager/modules/common/pom.xml

```diff
@@ -59,19 +59,6 @@
 
   <build>
     <plugins>
-
```

Comment:
Can we eventually put those changes in another commit?

Reply:
Done, but i need some time to check locally (builded successfully, but i need to install and check it by hands).

Reply:
Okay... it takes long time, but I checked)
Everything looks good!